tools/runqslower.py

```diff
@@ -150,7 +150,9 @@
     // output
     events.perf_submit(ctx, &data, sizeof(data));
 
-    start.delete(&pid);
+    //array map has no delete method, set ts to 0 instead
+    *tsp = 0;
+    start.update(&pid, tsp);
```

Comment:
Indeed, the array deleting elem operation will return -EINVAL.
In kernel/bpf/arraymap.c,
```
/* Called from syscall or from eBPF program */
static int array_map_delete_elem(struct bpf_map *map, void *key)
{       
        return -EINVAL;
}  
```

The above `start.update(...)` is not needed, `*tsp = 0` should already set ts to 0 in the array map.
The same for another case below.

Reply:
thanks! removed  the update.